src/measure_theory/outer_measure.lean

```diff
@@ -341,19 +351,22 @@ variables {s s₁ s₂ : set α}
 
 /-- A set `s` is Carathéodory-measurable for an outer measure `m` if for all sets `t` we have
   `m t = m (t ∩ s) + m (t \ s)`. -/
-private def C (s : set α) : Prop := ∀t, m t = m (t ∩ s) + m (t \ s)
+def caratheodory_sets (s : set α) : Prop := ∀t, m t = m (t ∩ s) + m (t \ s)
```

Comment:
For me `caratheodory_sets` suggests that this is a `set (set α)`. I think that `set α → Prop` should be named `caratheodory_set`, `is_caratheodory`, or `is_caratheodory_set`.